content/en/docs/Getting started/_index.md

```diff
@@ -88,37 +88,68 @@ To copy the example site:
 
 1.  Make a local working copy of the example site directly using `git clone`:
 
-    ```bash
-    git clone https://github.com/google/docsy-example.git
-    ```
-1. In GitHub, create a [new empty repo](https://help.github.com/en/articles/create-a-repo) for your site with your chosen repo name. For clarity you may also want to rename the root directory (`docsy-example`) of your local working copy to match, though everything will still work even if you don't.
-
-1. In your site root directory, change your local copy's `origin` to your new repo (otherwise you'll be trying to push changes to the Docsy example site rather than to your own site):
+     ```shell
+     git clone https://github.com/google/docsy-example.git
+     ```
+    
+1. Switch to the root of the cloned project, for example:
 
-    ```bash
-    $ git remote remove origin
-    $ git remote add origin https://github.com/my/example.git
-    $ git push -u origin master
-    ``` 
+    ```shell
+    cd docsy-example
+    ```
 
 1. Get local copies of the project submodules so you can build and run your site locally:
 
-    ```bash
-    $ git submodule update --init --recursive
+    ```shell
+    git submodule update --init --recursive
     ```
-1.  Switch to your site root directory, and build your site:
     
+1. Build your site:
+    
+    ```shell
+    hugo server
     ```
-    $ hugo server
-    ```
-1.  Preview your site: http://localhost:1313/.
     
+1. Preview your site in your browser at: http://localhost:1313/
+
+    Tip: Use `Ctrl + c` to stop the Hugo server.
+
+1. Now that you have a local copy running, you can a create a new repository and then `git push` your Docsy site:
+
+   1. [Create a new remote repository in GitHub](https://help.github.com/en/articles/create-a-repo) 
+      for your site with your chosen repo name. For clarity you may also want to rename the root 
+      directory (`docsy-example`) of your local working copy to match, though everything will still 
+      work even if you don't.
+
+   1. Configure your 
+      [remote (`origin`)](https://help.github.com/en/articles/configuring-a-remote-for-a-fork)
+      in your project:
```

Comment:
This is kind of weirdly phrased, you wouldn't say "configure your remote (origin)", especially as the important thing here is that you're *resetting*  "origin", configuring doesn't convey that there's one already there from cloning the example  - I also think having this as a step with substeps is overkill given that one of them is just verifying that it worked.

I think we can just have the next step (and the verification)

Reply:
removed all "local" and "remote" references, cleaned up "site" vs "repo" terminology, and consolidated unnecessary nested list